.github/scripts/install-cvc4.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Adapted from https://github.com/commercialhaskell/stack
```

Comment:
Does this file need to mention the above repo's license, and/or include it somehow? Guess it depends how much of a derivative work this is.

Reply:
it's a very simple script I doubt attribution is necessary